src/pages/Login/Login.jsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react'
-
+import { useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../../context/auth-context';
+import "./Login.css"
 export const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  
+  const { auth, setAuth } = useAuth();
+  
+  
+  const navigate = useNavigate('');
+
+  const loginHandler = async () => {
+    try {
+
+      const response = await axios.post('/api/auth/login', { email: 'adarshbalika@gmail.com', password: 'adarshbalika' });
+      // const response = await axios.post('/api/auth/login', { email, password });
+      console.log(response)
```

Comment:
this console.log for testing purpose it can be removed once done 

Please Check at  other places also there are many console.log